src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
 // export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  let partner;
+
+  if (isMarried) {
+    partner = sex === 'f' ? 'husband' : 'wife';
+  }
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+      {person.age && (
```

Comment:
if `age === 0` then the 0 will be displayed before the `p` tag.

```suggestion
      {person.age !== undefined && (
```